src/main/resources/static/api.yml

```diff
@@ -3740,42 +3706,62 @@ components:
             - extensions
             - departures
             - cancellations
-    Person:
+    PersonInfo:
```

Comment:
Does this have to be named `PersonInfo`? Would make life easier in the frontend if we could just call it `Person` 😃

Reply:
Initially it was so I could change each usage of `Person` in api.yml in a discreet commit.  Have added another commit to rename it back to Person now.

Reply:
Noice one!